domains/needrose.json

```diff
@@ -0,0 +1,12 @@
+{
+        "owner": {
+           "username": "Neeedrose",
+           "email": "",
+           "discord": "815732542695145473"
+        },
+    
+        "record": {
+            "A": ["34.125.251.89"]
```

Comment:
502 Gateway, are you able to provide a preview?

Reply:
Oh sorry, didn't know you needed my website to be up yet. I am still customizing the website. I'll make sure to tell you when it's up. Sorry for wasting your time.

Reply:
No it's all good, I'm going to close this PR, but you can hit one button to reopen it once you are done. Have a good day.